pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -340,7 +340,8 @@ https://github.com/sjl/gundo.vim/,,
 https://github.com/junegunn/gv.vim/,,
 https://github.com/m4xshen/hardtime.nvim/,HEAD,
 https://git.sr.ht/~sircmpwn/hare.vim,HEAD,
-https://github.com/ThePrimeagen/harpoon/,,
+https://github.com/ThePrimeagen/harpoon/,master,
+https://github.com/ThePrimeagen/harpoon/,harpoon2,harpoon2
```

Comment:
looks like harpoon2 also depends on plenary, could you add an override for that?

Reply:
Indeed ! Done :)